src/main/java/org/team1294/firstpowerup/robot/Robot.java

```diff
@@ -1,36 +1,43 @@
 package org.team1294.firstpowerup.robot;
 
-import edu.wpi.first.wpilibj.CameraServer;
 import edu.wpi.first.wpilibj.IterativeRobot;
+import edu.wpi.first.wpilibj.command.Command;
 import edu.wpi.first.wpilibj.command.Scheduler;
+import edu.wpi.first.wpilibj.smartdashboard.SendableChooser;
 import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
-import org.team1294.firstpowerup.robot.commands.DriveStraightCommand;
+import org.team1294.firstpowerup.robot.commands.AutoCenterPositionCommand;
+import org.team1294.firstpowerup.robot.commands.AutoDriveCommand;
+import org.team1294.firstpowerup.robot.commands.AutoSidePositionCommand;
 import org.team1294.firstpowerup.robot.commands.ResetEncoderCommand;
 import org.team1294.firstpowerup.robot.commands.ResetGyroCommand;
 import org.team1294.firstpowerup.robot.subsystems.*;
+import org.team1294.firstpowerup.robot.subsystems.VisionSubsystem;
 
 /**
- * The main class for the Robot. This handles the creation of the various
- * subsystems of the robot, and the timing of when events happen. Each robot
- * mode has an "init" and a "periodic" method. The "init" event happens one
- * time when the drivers' station (DS) starts an op-mode, and the "periodic"
- * event happens every time a packet is received from the DS (~20 ms).
+ * The main class for the Robot. This handles the creation of the various subsystems of the robot,
+ * and the timing of when events happen. Each robot mode has an "init" and a "periodic" method. The
+ * "init" event happens one time when the drivers' station (DS) starts an op-mode, and the
+ * "periodic" event happens every time a packet is received from the DS (~20 ms).
  */
 public class Robot extends IterativeRobot {
+
     public static DriveSubsystem driveSubsystem;
+    public static VisionSubsystem visionSubsystem;
     public static ClimbSubsystem climbSubsystem;
     public static CelescoPotTestSubsystem celescoPotTestSubsystem;
```

Comment:
I probably should have done this in #9, but can you remove `CelescoPotTestSubsystem`? It's no longer necessary since the potentiometer is implemented in one of the other subsystems already.

Reply:
will do